chapters/equations.tex

```diff
@@ -788,77 +778,63 @@ \section{Initialization, initial equation, and initial algorithm}\doublelabel{in
 that need to be set.
 \end{nonnormative}
 
-All variables declared as parameter having \lstinline!fixed = false! are treated as
-unknowns during the initialization phase, i.e.\ there must be additional
-equations for them -- and the start-value can be used as a guess-value
-during initialization.
+All variables declared as \lstinline!parameter! having \lstinline!fixed = false! are treated as unknowns during the initialization phase, i.e.\ there must be additional equations for them --- and
+the \lstinline!start!-value can be used as a guess-value during initialization.
 
 \begin{nonnormative}
 In the case a parameter has both a binding equation and \lstinline!fixed = false! a diagnostics is recommended, but the parameter should be solved from the binding equation.
 
 Non-discrete (that is continuous-time) \lstinline!Real! variables \lstinline!vc! have exactly one initialization value since the rules above assure that during initialization
-\lstinline!vc = pre(vc) = vc.startExpression! (if \lstinline!fixed=true!).
+\lstinline!vc = pre(vc) = vc.startExpression! (if \lstinline!fixed = true!).
 
 Before the start of the integration, it must be guaranteed that for all variables \lstinline!v!, \lstinline!v = pre(v)!. If this is not the case for some variables
 \lstinline!vi!, \lstinline!pre(vi) := vi! must be set and an event iteration at the initial time must follow, so the model is re-evaluated, until this condition is fulfilled.
 
-A Modelica translator may first transform the continuous equations of a model, at least conceptually, to state space form. This may require to differentiate equations for index
+A Modelica translator may first transform the continuous equations of a model, at least conceptually, to state space form.  This may require to differentiate equations for index
 reduction, i.e., additional equations and, in some cases, additional unknown variables are introduced.  This whole set of equations, together with the additional constraints
 defined above, should lead to an algebraic system of equations where the number of equations and the number of all variables (including \lstinline!der! and \lstinline!pre!
 variables) is equal.  Often, this is a nonlinear system of equations and therefore it may be necessary to provide appropriate guess values (i.e., \lstinline!start! values and
-\lstinline!fixed=false!) in order to compute a solution numerically.
+\lstinline!fixed = false!) in order to compute a solution numerically.
 
-It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index. A tool may add or remove initial equations
+It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index.  A tool may add or remove initial equations
```

Comment:
This looks like adding a second space before "A"; why?

Reply:
Right, that's an unrelated change purely for source code readability.  I'll remove it if you don't like it.

Reply:
Please do.

Reply:
Fixed.